src/app/services/culture.service.ts

```diff
@@ -1,17 +1,35 @@
 import { Injectable } from '@angular/core';
-import { HttpClient } from '@angular/common/http';
-import { Details } from '../models/card';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
 import { Observable } from 'rxjs';
+import { Details } from '../models/card';
 
 @Injectable({
   providedIn: 'root'
 })
-
 export class CultureService {
-  private jsonUrl = 'assets/data/culture.json';
+
+  private apiUrl = 'http://localhost:8080/api/culture/getdata';
```

Comment:
wrong , you should not use localhost here
. We are hosting it and you need domain name